microsoft/botframework-solutions
# [C#] Add YAML files in VSIX Generators

Add to the Virtual Assistant Template and Skill Template projects the pipeline folder with the YAML file configuration to generate the samples. 

## Description

### For Template and Sample projects
- Add `Pipeline` folder with the `YAML` file configuration.

## Testing Steps
1. Go to `botframework-solutions\templates\Skill-Template\csharp`.
1. Check that inside Template and Sample folder exist the Pipeline folder with the `YAML` configuration.
1. Go to `botframework-solutions\templates\Virtual-Assistant-Template\csharp`.
1. Check that inside `Template` and `Sample` folder exist the `Pipeline` folder with the `YAML` configuration.
1. Check that running the `VSIX` generator in both cases, the solution generates the samples correctly with the `Pipeline` folder.

## Checklist
*-*
